test/hs/Test/Ganeti/HTools/Backend/Text.hs

```diff
@@ -276,7 +279,12 @@ prop_CreateSerialise =
        Ok (_, _, _, [], _) -> counterexample
                               "Failed to allocate: no allocations" False
        Ok (_, nl', il, _, _) ->
-         let cdata = Loader.ClusterData defGroupList nl' il ctags
+         let
+             -- makeSmallCluster created an empty cluster, that had some instances allocated,
```

Comment:
By changing the behaviour of the calculations I broke this test. Now the fMem (state-of-world) isn't updated during placement simulations (as it doesn't make sense subtracting state-of-record instance sizes from the state-of-world free memory for reasons explained in the comments) - rather it is considered invalid (outdated).